dfhack-config/init/onMapLoad.init

```diff
@@ -0,0 +1,5 @@
+# This file runs when a fortress map is loaded, after any onLoad.init files
```

Comment:
I think this and onMapUnload apply to adventure mode too, although I could be wrong (depends on whether world.map is populated by DF)

Reply:
Updated comments to reflect this